ERCS/eip-draft_puppet_proxy_contract.md

```diff
@@ -0,0 +1,342 @@
+---
```

Comment:
```suggestion
---
eip: 7613
```

Reply:
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.